### PR TITLE
Added performance test

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

**Review by sfc-gh-jsikorski — APPROVED** [review]

File: .github/workflows/performance_test.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          python-version: "3.11"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+      - name: Run integration tests
+        env:
+          PYTHONPROFILEIMPORTTIME: "1"
+        run:
+          time snow --help
```
Comment (sfc-gh-turbaszek) [inline .github/workflows/performance_test.yaml:30]: I think this is more of a check than test, we don't compare with anything right? Maybe we should make it a normal pytest test? We can run snowcli via `subprocess` and measure the time. This gives us more elasticity in saving the results as well as comparing them

File: tests/teset_performance.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import subprocess
```
Comment (sfc-gh-turbaszek) [inline tests/teset_performance.py:1]: There's typo in file name

File: tests/teset_performance.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+from timeit import default_timer as timer
+
+SAMPLE_AMOUNT = 20
+EXECUTION_TIME_THRESHOLD = 0.55
+
+
+def test_snow_help_performance():
+    results = []
+    for i in range(0, SAMPLE_AMOUNT):
```
Comment (sfc-gh-turbaszek) [inline tests/teset_performance.py:10]:
```suggestion
    for i_ in range(0, SAMPLE_AMOUNT):
```

As we don't use `i`

File: tests/teset_performance.py (hunk 2 above)
Comment (sfc-gh-astus) [reply tests/teset_performance.py]: fixed

File: tests/teset_performance.py (hunk 3 above)
Comment (sfc-gh-astus) [reply tests/teset_performance.py]: changed

**Review by sfc-gh-turbaszek — APPROVED** [review]

**Review by sfc-gh-turbaszek — APPROVED** [review]

**Pull request merged**